samples/bluetooth/mesh/light_dimmer/README.rst

```diff
@@ -117,7 +117,10 @@ Button 2:
    On long press and release, **Button 2** will publish a Scene Store message using the configured publication parameters of its model instance, and store the current LED state of all the targets under the scene with the most recently recalled scene number.
 
 .. note::
-   :ref:`zephyr:thingy53_nrf5340` and the :ref:`zephyr:nrf52840dongle_nrf52840` only support dimming and the on/off functionality, not the scene selection functionality.
+   The :ref:`zephyr:nrf52840dongle_nrf52840` only support dimming and the on/off functionality, not the scene selection functionality.
```

Comment:
```
   .. note::
      On the :ref:`zephyr:nrf52840dongle_nrf52840`, the scene selection functionality will not be available as the device only has one button.
```
Indented to be inside the button description, and used the same wording as the earlier note.

Could also move this note to be at the start of the User interface section so we don't have two notes back-to-back. In that case, I'd also add the second line from the earlier note for completeness' sake.

Reply:
Going by your first suggestion, should both notes be indented then?
Or by your second suggestion, this note will be more or less identical to the one in the Overview then. Is that a good thing?

Reply:
Ah, yes, Thingy:53 note (and looking at it, the note under LEDs) should be indented.

I just checked the built documentation for the PR, and it seems that if the nRF54840 Dongle note is moved to the start of User interface, it might be visible on the screen at the same time as the one in Overview. You are right, that is needless duplication. Considering that, I would suggest keeping the note where it is with just one line, and changing the note for Thingy:53 to a tip instead of note to differentiate it a bit. In this case we can remove "remember that" from the line as well so it's just:
```
      On Thingy:53, **Button 2** can be accessed by removing the top part of the casing.
```